lab_4_summarization_textrank/main.py

```diff
@@ -84,32 +104,57 @@ def __init__(self, stop_words: tuple[str, ...], punctuation: tuple[str, ...]) ->
         """
         Constructs all the necessary attributes
         """
-        pass
+        super().__init__(stop_words, punctuation)
```

Comment:
consider validating arguments before passing them to the parent class, not the other way around